docs/release_guide.md

```diff
@@ -13,13 +13,16 @@ This guide describes the process to create a GitHub Release for this project.
 Once an RC has been found to be stable, cut a release tagged `vX.Y.Z` using the following steps.
 
 - [Release Guide](#release-guide)
-  - [Release Candidates](#release-candidates)
-  - [Create a release branch](#create-a-release-branch)
+  - [Create release branches](#create-release-branches)
```

Comment:
If we can parameterize the builds enough to build FIPS and non-FIPS from the same source and utilize that in CI and use something like a `strategy.matrix` to do both, I can see where we shouldn't need to change the release guide at all.

Reply:
+1

Reply:
We would still need to publish different tags/images, but in general I agree if we can parameterize this to that degree.

Reply:
Looks like this file needs another pass.

Reply:
Ah yeah; I don't think there needs to be any changes in here after all right? Except maybe a note that ARM64 images will only be available via MCR